users/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import requests, jwt
+from datetime    import datetime, timedelta
 
-# Create your views here.
+from django.shortcuts import redirect
+from django.views import View
+from django.http  import JsonResponse
+from django.conf  import settings
+
+from users.models import User
+
+class KakaoLogin(View):
+    def get(self, request):
+        REST_API_KEY = settings.REST_API_KEY
+        REDIRECT_URI = 'http://10.58.7.139:8000/users/kakao/callback'
+        
+        API_HOST = f'https://kauth.kakao.com/oauth/authorize?client_id={REST_API_KEY}&redirect_uri={REDIRECT_URI}&response_type=code'
+
+        return redirect(API_HOST)
+
+
+class GetUserInfoView(View):
+    def get(self, request):
+        try:
+            code            = request.GET.get('code')
+            kakao_token_api = 'https://kauth.kakao.com/oauth/token'
+            data = {
+                'grant_type': 'authorization_code',
+                'client_id': 'c5bd4a71d71645a01317f6f9d091193d',
+                'code': code
+            }
+            response       = requests.post(kakao_token_api, data = data)
```

Comment:
requests 모듈을 사용하실 때에는 `timeout` 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 발생 합니다. timeout 옵션을 추가해 주세요.
```suggestion
            response       = requests.post(kakao_token_api, data = data, timeout = ..)
```